dm/master/server.go

```diff
@@ -134,12 +136,12 @@ func (s *Server) Start() error {
 	go func() {
 		err2 := s.svr.Serve(grpcL)
 		if err2 != nil && !common.IsErrNetClosing(err2) && err2 != cmux.ErrListenerClosed {
-			log.Errorf("[server] gRPC server return with error %s", err2.Error())
+			log.L().Error("fail to start gRPC server", zap.Error(err2))
 		}
 	}()
 	go InitStatus(httpL) // serve status
 
-	log.Infof("[server] listening on %v for gRPC API and status request", s.cfg.MasterAddr)
+	log.L().Info(" listening gRPC API and status request", zap.String("address", s.cfg.MasterAddr))
```

Comment:
how about define a variable `logger` in the `master`, and set logger with prefix `server` in `init` function?

Reply:
what's the definition of `server`?

Reply:
<img width="487" alt="image" src="https://user-images.githubusercontent.com/5793595/59416810-b8012780-8df8-11e9-810d-0108c6aa1d0d.png">

Reply:
like this in my pr https://github.com/pingcap/dm/pull/170

Reply:
I choose to ignore `server` label  to indicate  `I'm main process`, but I prefer to add `component` for other independent component